config/OPM-Reconfig.cfg

```diff
@@ -0,0 +1,499 @@
+// Original draft 2019/02/20 by Eridan/HSJasperism
+
+// Removes the Asteroid configs for Kopernicus
+// Kopernicus will still load its ScenarioModule named DiscoverableObjects after killing stock ScenarioDiscoverableObjects
+// but void Start()'s foreach loop will not go though anything because no Asteroid configs will exist.
+
+@Kopernicus:AFTER[OPM]:NEEDS[CustomAsteroids]
+{
+	!Asteroid[Stock] {}
+	!Asteroid[OPMSarnus] {}
+	!Asteroid[OPMUrlum] {}
+	!Asteroid[OPMNeidon] {}
+}
+
+// Custom Asteroid Configs
+// Jool, Sarnus, Urlum, & Neidon configs based on Artifexian's tutorials on Gas Giant systems
+// Sarnus, Urlum, & Neidon attempt to mimic JadOfMaar's configs
+
+AsteroidSets:NEEDS[CustomAsteroids]
+{
+	name = OPM_GasGiantObjects
+
+ 	ASTEROIDGROUP
+	{
+		name = joolInner
+		title = #autoLOC_CustomAsteroids_OPM_GroupInnerObj
+
+ 		centralBody = Jool
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Jool.rad, 1.89)
+			max = Ratio(Jool.rad, 2.44)
+		}
+
+ 		eccentricity
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		inclination
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
```

Comment:
Do you want these moonlets to appear from the start of the game? There's a `detectable` block that lets them appear only if the planet has been visited (like for stock [Drestroids](https://github.com/Starstrider42/Custom-Asteroids/blob/master/config/Stockalike.cfg#L74-L80)), only if there's a spacecraft in orbit, etc. If you're interested, you can find a complete list of conditions [in the documentation](http://starstrider42.github.io/Custom-Asteroids/newbelts.html#adv_groups).

Reply:
I've added a series of detectable {} module manager patches. I prefer this for now, at least so turning it on or off for testing in fresh saves is easier.